frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -425,8 +633,8 @@ const AttendanceRow = React.memo(function AttendanceRow({
   const updateValue = useCallback(
     (date: LocalDate, rangeIx: number, updatedRange: TimeRange) => {
       setValues(
-        values.map((val) => {
-          return val.date === date
+        values.map((val) =>
+          val.date === date
```

Comment:
Is this object instance equal check intentional?

Reply:
I didn't make this check (just removed the redundant braces), but I guess it might be? It's called from the editor component's update callback, which has the exact reference to the date that needs to be updated, so it'll find the exact entry that was requested to be updated.